integration_tests/steps/integration_tests.py

```diff
@@ -203,14 +203,12 @@ def running_the_simulation(context):
     paused = False
     pause_after = duration()
     repl = False
-    export = False
+    no_export = True
```

Comment:
I haven't looked at the d3a-web PR so this might already be handled, but we need to also check that the d3a on the production server does not export simulation results, because this could lead to filling up the disk space of the d3a simulation pod.

Reply:
Yes you are right, i tried to deal with that by added this line to the d3a-web.py:
`kwargs = {"no_export": True}`